playwright/src/test/java/com/microsoft/playwright/TestTracing.java

```diff
@@ -284,41 +357,95 @@ public void shouldNotRecordNetworkActions(@TempDir Path tempDir) throws IOExcept
     Path traceFile1 = tempDir.resolve("trace1.zip");
     context.tracing().stop(new Tracing.StopOptions().setPath(traceFile1));
 
-    List<TraceEvent> events = parseTraceEvents(traceFile1);
-    List<String> calls = events.stream().filter(e -> e.renderedTitle() != null).map(e -> e.renderedTitle())
-      .collect(Collectors.toList());
-    assertEquals(asList("Frame.goto"), calls);
+    showTraceViewer(traceFile1, traceViewer -> {
+      assertThat(traceViewer.actionTitles()).hasText(new Pattern[] {
+        Pattern.compile("Navigate to \"/empty.html\"")
+      });
+    });
+  }
+
+  private void showTraceViewer(Path tracePath, TraceViewerConsumer callback) throws Exception {
+    Path driverDir = Driver.ensureDriverInstalled(java.util.Collections.emptyMap(), true).driverDir();
```

Comment:
Move this method along with the trace viewer POM into its own file? Also consider turning the viewer into a fixture.